pylds/tools.py

```diff
@@ -9,40 +9,38 @@
 import matplotlib.pyplot as plt
 import ipywidgets as widgets
 
-def draw_lagrangian_descriptor(LD, LD_type, grid_parameters, tau, p_value, colormap_name='bone', colormap_mode=1, interactive=False):
+def ld_plot(LD, LD_gradient, grid_parameters, colormap, interactive, string_title, colormap_gradient):
     """
     Draws a Lagrangian descriptor contour plot and a contour plot showing the magnitude of its gradient field.
 
     Parameters
     ----------
     LD : ndarray, shape(n, )
         Array of Lagrangian Descriptor values.
-    
-    LD_type : str
-        Type of LD to plot. Options: 'forward', 'backward', 'total'.
-    
+
+    LD_gradient : ndarray, shape(n, )
+        Array of Lagrangian Descriptor gradient values.
+
     grid_parameters : list of 3-tuples of floats
         Limits and size of mesh per axis.
-    
-    tau : float
-        Upper limit of integration.
-        
-    p_value : float
-        Exponent in Lagrangian descriptor definition.
-    
-    norm : bool, optional
-        True normalises LD values.
-    
-    colormap_name : str, optional
-        Name of matplotlib colormap for plot.
-        
+
+    colormap : str, optional
+        Name of matplotlib colormap for contouor plot.
+
     interactive : bool
-        True allows interactively adjucting the gradient plot minimum.
-    
+        True allows interactively adjusting the gradient contouor plot minimum and maximum.
+
+    string_title : string
+        Plot title.
+
+    colormap_gradient : string
+        Name of matplotlib colormap for gradient contouor plot.
+
     Returns
     -------
         Nothing.
```

Comment:
Not sure `Nothing` is appropriate.
There are no guidelines for graphical outputs in here
https://numpydoc.readthedocs.io/en/latest/format.html 

But, in the documentation of [`matplotlib.pyplot.plot`](https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.plot.html) or [`matplotlib.axes.Axes.contourf`](https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.axes.Axes.contourf.html?highlight=contourf#matplotlib.axes.Axes.contourf) they provide a brief description of the graphical output and/or point out to some `matplotlib` object.

Reply:
I see what you mean. The matplotlib functions you mention return an object, in contrast to ours. Would `No output` or similar work?

Reply:
@Shibabrat any thoughts?

Reply:
Done